rohmu/__init__.py

```diff
@@ -44,8 +46,29 @@ def get_class_for_transfer(obj_store: Config) -> Type[BaseTransfer]:
     raise InvalidConfigurationError("unsupported storage type {0!r}".format(storage_type))
 
 
+def get_class_for_notifier(notifier_config: dict) -> Type[Notifier]:
+    notifier_type = notifier_config[NOTIFIER_TYPE]
+    if notifier_type == "http":
+        from .notifier.http import BackgroundHTTPNotifier
+
+        return BackgroundHTTPNotifier
+    raise InvalidConfigurationError("unsupported storage type {0!r}".format(notifier_type))
+
+
+def get_notifier(notifier_config: dict) -> Notifier:
```

Comment:
Do we want  `mypy` to complain on bare generics?

Reply:
what do you mean? `Notifier` should be an `ABC` not a `TypeVar`.

Reply:
Sorry, I meant for the `dict`, in my brain I had specified it :-D

Reply:
AH, I tried to do a best effort typing on this PR. So I wasn't really trying to chase what are the correct types for the values. What do you suggest?

Reply:
I don't suggest anything :-p I think it's just I'm used to `mypy` moaning about this in other projects I guess 😓